boards/airfy-beacon/include/board.h

```diff
@@ -28,7 +28,7 @@
 #endif
 
 /**
- * @brief   Xtimer configuration
+ * @name   Xtimer configuration
```

Comment:
Unfortunately, it's not that simple since this rule can change other valid `@brief`